### PR TITLE
evra's game 05-20-20

### PR DESCRIPTION
**Pull request by oscillator3**

### CODE REVIEW

**Comment by dicksontsai** [comment]
Your current PR deletes a game. Please remove that change.

**Comment by Devking** [comment]
`.DS_Store` should also be added to `.gitignore`

**Comment by oscillator3** [comment]
restored dickson20200516.json

**Comment by Devking** [comment]
🚢

File: evra20200520.json [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+        },
+        {
+          "value": 800,
+          "text": "After World War II, the US installed a new government with just the right ingredients for this.",
+          "answer": "a post-war economic miracle",
+          "isDailyDouble": false
+        },
+        {
+          "value": 1000,
+
```
Comment (dicksontsai) [inline evra20200520.json:177]: There's an extra line here.

**Pull request merged**